### PR TITLE
Update nwbib-spatial from Wikidata

### PR DESCRIPTION
**Pull request by fsteeg**
See https://github.com/hbz/nwbib/issues/542 & https://github.com/hbz/nwbib#classification

### CODE REVIEW

File: nwbib/nwbib-spatial.ttl [hunk 1]
```diff
@@ -3866,7 +3859,7 @@ nwbib-spatial:Q1541601
 
 nwbib-spatial:Q1906190
         a               skos:Concept ;
-        skos:broader    nwbib-spatial:Q512233 ;
+        skos:broader    nwbib-spatial:Q59138859 ;
```
Comment (acka47) [inline nwbib/nwbib-spatial.ttl]: This is strange, as Q512233 is the ID and Q59138859 is redirected to it, see https://www.wikidata.org/entity/Q59138859.

File: nwbib/nwbib-spatial.ttl [hunk 2]
```diff
@@ -12688,7 +12695,7 @@ nwbib-spatial:Q1803277
 
 nwbib-spatial:Q432818
         a               skos:Concept ;
-        skos:broader    nwbib-spatial:Q512233 ;
+        skos:broader    nwbib-spatial:Q59138859 ;
```
Comment (acka47) [inline nwbib/nwbib-spatial.ttl]: Same as above

**Comment by fsteeg** [comment]
I re-generated from Wikidata again and that seems to fix it.

Maybe caused by cached data in the SPARQL service or something?

**Comment by acka47** [comment]
+1

**Pull request merged**